### PR TITLE
Velero docs update

### PR DESCRIPTION
**Pull request by talha0324**

### CODE REVIEW

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-114-dde83bce`

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-114-9a67cca8`

File: content/sre/backup-restore/stateful-app-example.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+~~~
+
+If everything goes well, your output should resemble to the one record above.
+
+## Take backup and destroy app
+---------
+
+To take velero backup, use velero command:
+~~~
+velero backup create cassandra-backup --include-namespaces cassandra-app
```
Comment (rasheedamir) [inline content/sre/backup-restore/stateful-app-example.md:207]: How do I get the velero CLI?

File: content/sre/backup-restore/stateful-app-example.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+~~~
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: cassandra-backup-restore
+  namespace: openshift-velero
+spec:
+  backupName: cassandra-backup
+  excludedResources:
+    - nodes
```
Comment (rasheedamir) [inline content/sre/backup-restore/stateful-app-example.md:244]: nodes?

File: content/sre/backup-restore/backup-restore.md [hunk 3]
```diff
@@ -1,11 +1,41 @@
...
+
+#### Manifests Backup
+
+| Namespaces       | Backup Frequency | Backup Retention |
+| ---------------- | ---------------- | ---------------- |
+|  All Namespaces  |    Every 6 hrs   | Last 12 Backup(s)   | 
+
+#### Volumes Backup
+
+Stakater only takes backups of the managed applications:
```
Comment (rasheedamir) [inline content/sre/backup-restore/backup-restore.md:30]: @usamaahmadkhan @talha0324 this doesn't seem to be correct

File: content/sre/backup-restore/stateful-app-example.md (hunk 2 above)
Comment (talha0324) [reply content/sre/backup-restore/stateful-app-example.md:244]: This is a default definition produced by velero CLI

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-114-e48a9fb3`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@talha0324 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-114-2ac4fc74`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.59`

**Pull request merged**